src/main/scala/io/archivesunleashed/package.scala

```diff
@@ -41,17 +45,52 @@ import scala.util.matching.Regex
 package object archivesunleashed {
   /** Loads records from either WARCs, ARCs or Twitter API data (JSON). */
   object RecordLoader {
+
+    val log: Logger = Logger.getLogger(getClass.getName)
+
+    /** Gets all archive files by applying filters prefix, numFiles and maxSize
+      *
+      * @param dir the path to the directory containing archive files
+      * @param fs filesystem
+      * @param prefix prefix of archive files
+      * @param numFiles number of archive files
```

Comment:
`numFiles` is used for debugging only. For instance we can do a sanity check by running on only 10 files by setting `numFiles = 10`.

We can also limit the maximum file size since I found out that large files are causing the heartbeat timeout issue when I run AUT on those ARC files.

Reply:
@ruebot but how do you determine the correct value for `spark_heartbeat_interval`? by trial-and-error?

Reply:
FWIW I determined by trial-and-error. Since it only affects a small number of (W)ARCs, putting the number relatively high was fine in our particular application. We have it set to `600s` by default right now.